resources/route53-health-checks.go

```diff
@@ -16,24 +16,33 @@ func ListRoute53HealthChecks(sess *session.Session) ([]Resource, error) {
 	params := &route53.ListHealthChecksInput{}
 	resources := make([]Resource, 0)
 
-	resp, err := svc.ListHealthChecks(params)
-	if err != nil {
-		return nil, err
-	}
+	getHealthChecks := func() *bool {
+		b := true
+		return &b
+	}()
+
+	for *getHealthChecks == true {
+		resp, err := svc.ListHealthChecks(params)
+		if err != nil {
+			return nil, err
+		}
 
-	for _, check := range resp.HealthChecks {
-		resources = append(resources, &Route53HealthCheck{
-			svc:  svc,
-			id:   check.Id,
-		})
+		for _, check := range resp.HealthChecks {
+			resources = append(resources, &Route53HealthCheck{
+				svc: svc,
+				id:  check.Id,
+			})
+		}
+		getHealthChecks = resp.IsTruncated
```

Comment:
Maybe `getHealthChecks` should be called `isTruncated` for a easier understanding of the code.

Reply:
I looked how you handled these elsewhere and refactored to an unconditional while loop.